### ISSUE TITLE
Log lines are corrupted with random newlines

### ISSUE THREAD

**Issue by nbolton** [symless/synergy-core#6369]
Maybe related: #6374

### Operating Systems ###

All

### Synergy Version ###

1.10.0

### Command Line ###

Config app

### Steps to reproduce bug ###

1. Experience some sort of connection error
2. Check the log window
3. See that some log lines have lots of random spaces

```
[2018-07-24T15:30:50] NOTE: server is dead
[
2018-07-24T15:
3
0:50]
 
WARNI
N
G: fai
l
ed to
 
connec
t
 to server: s
e
rver
 is 
n
ot responding
[2018-07-24T15:30:51] NOTE: connecting to '192.168.7.5': 192.168.7.5:24800
```

**Comment by nbolton** [comment]
Closing tentatively. Possibly fixed by #6374

**Comment by nbolton** [comment]
Saw this again today, probably not fixed by #6374

**Comment by Jnewbon** [comment]
Unable to reproduce on v1.10.2, possibly fixed. Closing till the issue it is seen again.
